src/helpers/pivot/pivot_menu_items.ts

```diff
@@ -15,3 +15,30 @@ export const pivotProperties: ActionSpec = {
   },
   icon: "o-spreadsheet-Icon.PIVOT",
 };
+
+export const PIVOT_SWITCH_STATIC: ActionSpec = {
+  name: _t("Fix pivot formulas"),
+  execute(env) {
+    const position = env.model.getters.getActivePosition();
+    const cell = env.model.getters.getCorrespondingFormulaCell(position);
```

Comment:
you don't have to do this check as it's already done the the following getter. If we consider that it's better with the double check, I would have an early return if `!cell` to avoid useless computations.